munit-scalacheck/shared/src/main/scala/munit/ScalaCheckSuite.scala

```diff
@@ -34,21 +35,37 @@ trait ScalaCheckSuite extends FunSuite {
 
   protected def scalaCheckPrettyParameters = Pretty.defaultParams
 
+  // Allow specifying an initial seed in its Base64 encoded form
+  implicit class XtensionScalaCheckTestParameters(
+      params: ScalaCheckTest.Parameters
+  ) {
+    def withInitialSeed(encodedSeed: String): ScalaCheckTest.Parameters =
+      params.withInitialSeed(Seed.fromBase64(encodedSeed).get)
+  }
```

Comment:
ScalaCheck does something analogous in `propertyWithSeed`, and I think it makes sense (since we print the Base64 encoded form in test failures.